schematics/types/compound.py

```diff
@@ -244,6 +244,13 @@ def _export(self, list_instance, format, context):
 class DictType(CompoundType):
     """A field for storing a mapping of items, the values of which must conform to the type
     specified by the ``field`` parameter.
+    
+    Use it like this:
+    
+    .. code-block:: python
```

Comment:
Yes - but only if Python is the default highlighted language. I usually prefer to be explicit. Let me know if you feel strongly about this and I'll make the change.